src/components/wizard-completion-step.component.css

```diff
@@ -1,4 +1,4 @@
-:host {
+wizard-completion-step {
```

Comment:
Is this required, I mean we don't change `wizard-step.component.css` to start with `wizard-step`, while we do this here?